python/paddle/fluid/dygraph/dygraph_to_static/program_translator.py

```diff
@@ -371,17 +371,24 @@ def get_concrete_program(self, *args, **kwargs):
         Returns:
             Traced ConcreteProgram and executable translated Layer.
         """
-        # 1. unify args/kwargs and replace Tensor with InputSpec
+        # 1. verify the instance is initialized in imperative mode.
+        if isinstance(self._class_instance, layers.Layer):
+            if not self._class_instance._init_from_imperative_mode:
+                raise RuntimeError(
+                    " `paddle.jit.to_static` is only available in imperative mode. Please add `paddle.disable_static()` in front of your code."
```

Comment:
we will use `dynamic mode` and `static mode`, so maybe here `dynamic mode` is better

Reply:
Thanks, done